fdbsyncd/fdbsync.cpp

```diff
@@ -509,9 +659,6 @@ void FdbSync::macAddVxlan(string key, struct in_addr vtep, string type, uint32_t
     fvVector.push_back(t);
     fvVector.push_back(v);
 
-    SWSS_LOG_INFO("%sVXLAN_FDB_TABLE: ADD_KEY %s vtep:%s type:%s", 
```

Comment:
Why is this removed?

Reply:
@prsunny : Thanks, Done.  this must have got removed while resolving conflicts or rebase and merge.